1-conan-export.ps1

```diff
@@ -0,0 +1,8 @@
+$Cd = Get-Location
```

Comment:
Developer PowerShell for VS 2019 (Windows 10)
```
.\1-conan-export.ps1 : File C:\projects\synfig-conan\synfig\1-conan-export.ps1 cannot be loaded because running scripts is disabled on this system. For more information, see about_Execution_Policies at
https:/go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:1
+ .\1-conan-export.ps1
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : SecurityError: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess
```

Fixed by:
```
set-executionpolicy remotesigned -Scope CurrentUser
```

Maybe add this as a preparing step before build?